app/services/certificate_finder_service.rb

```diff
@@ -6,7 +6,7 @@ def initialize(type, code, description)
   end
 
   def call
-    return [] if description_types_and_codes.empty? && (type.blank? || code.blank?)
+    return [] if description_types_and_codes.empty? && type.blank? && code.blank?
```

Comment:
just to say, the logic is different, but I guess is what you want

Reply:
Yeah, this is correct - I want the api to do the control of what gets validated and just let users in scripts, etc, be able to filter on any of these fields